cmd/kubelet/app/options/globalflags.go

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/pflag"
+
+	// libs that provide registration functions
+	"k8s.io/apiserver/pkg/util/logs"
+	"k8s.io/kubernetes/pkg/version/verflag"
+
+	// ensure libs have a chance to globally register their flags
```

Comment:
I think this broke cross-build - https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-cross-build/

Reply:
diff on master between last green and first failed:
```
$ git diff b9a62423c6215a154eda712d91f6dcd83403e236 10a98ef1ea26a1426ecf4a0d23722955215f8fdd --numstat
0       1       cmd/kubelet/BUILD
0       13      cmd/kubelet/app/options/BUILD
0       167     cmd/kubelet/app/options/globalflags.go
6       25      cmd/kubelet/kubelet.go
1       1       pkg/credentialprovider/azure/azure_credentials.go
1       5       pkg/credentialprovider/gcp/jwt.go
2       2       pkg/kubelet/eviction/eviction_manager.go
1       7       pkg/version/verflag/verflag.go
0       13      pkg/volume/azure_dd/attacher.go
2       13      pkg/volume/azure_dd/azure_mounter.go
1       9       staging/src/k8s.io/apiserver/pkg/util/logs/logs.go
1       1       test/e2e/BUILD
1       1       test/e2e/e2e.go

```

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/58094